File: el_aventurero/src/main/java/com/cloud_technological/el_aventurero/dto/detalle_compras/CreateDetalleCompraDto.java
package com.cloud_technological.el_aventurero.dto.detalle_compras;

import java.math.BigDecimal;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class CreateDetalleCompraDto {
    private Long producto_id;
    private Integer cajas;
    private Integer unidades_por_caja;
    private BigDecimal costo_total;
    private BigDecimal precio_sugerido;
    private BigDecimal precio_venta;
}

File: el_aventurero_front/src/app/core/models/compra/create-detalle-compra.dto.ts
export interface CreateDetalleCompraDto {
  producto_id: number;
  cajas?: number;
  unidades_por_caja?: number;
  cantidad: number;
  costo_total: number;
  costo_unitario?: number;
  precio_sugerido?: number;
  precio_venta?: number;
  tipo?: string;
}
